sherlock/resources/data.json

```diff
@@ -844,6 +845,14 @@
     "username_claimed": "jenny",
     "username_unclaimed": "noonewouldeverusethis7"
   },
+  "Jupyter Community Forum": {
+    "errorMsg": "Oops! That page doesn’t exist or is private.",
+    "errorType": "message",
+    "url": "https://discourse.jupyter.org/u/{}/summary",
+    "urlMain": "https://discourse.jupyter.org"
+    "username_claimed": "choldgraf",
```

Comment:
Looks like you forgot to add a comma at the end of this line

Reply:
Done!